### PR TITLE
♿ A11Y Fixes

### PR DESCRIPTION
**Pull request by jssee**
Addressing the points made in issue #75 to have better semantics and help with A11Y

Old language selector vs new:
**Before**
before

**After**
after

### CODE REVIEW

**Comment by wojtekmaj** [comment]
This is significantly better. Please merge this!

**Comment by jssee** [comment]
@DeMoorJasper Updated

File: src/i18n/en/index.html [hunk 1]
```diff
@@ -32,17 +34,17 @@ <h2>Blazing fast, zero configuration web application bundler</h2>
...
-        </ul>
-      </div>
+      <form class="languages">
+        <select class="language-dropdown" onchange="location = this.options[this.selectedIndex].value;">
+          <option class="en" value="https://parceljs.org">🇺🇸 English</option>
+          <option class="pt" value="https://pt.parceljs.org">🇧🇷 Português</option>
+          <option class="ko" value="https://ko.parceljs.org">🇰🇷 한국어</option>
+          <option class="pl" value="https://pl.parceljs.org">🇵🇱 Polskie</option>
+          <option class="ru" value="https://ru.parceljs.org">🇷🇺 русский</option>
+          <option class="zh" value="https://zh.parceljs.org">🇨🇳 俄语</option>
```
Comment (dear-lizhihua) [inline src/i18n/en/index.html]: should be 中文, not 俄语.

File: src/i18n/es/index.html [hunk 2]
```diff
@@ -32,17 +34,17 @@ <h2>Empaquetador de aplicaciones web ultra-rápido, sin configuración</h2>
...
-      </div>
+      <form class="languages">
+        <select class="language-dropdown" onchange="location = this.options[this.selectedIndex].value;">
+          <option class="es" value="https://es.parceljs.org">🇪🇸 Español</option>
+          <option class="en" value="https://parceljs.org">🇺🇸 English</option>
+          <option class="pt" value="https://pt.parceljs.org">🇧🇷 Português</option>
+          <option class="ko" value="https://ko.parceljs.org">🇰🇷 한국어</option>
+          <option class="pl" value="https://pl.parceljs.org">🇵🇱 Polskie</option>
+          <option class="ru" value="https://ru.parceljs.org">🇷🇺 русский</option>
+          <option class="zh" value="https://zh.parceljs.org">🇨🇳 俄语</option>
```
Comment (dear-lizhihua) [inline src/i18n/es/index.html]: should be 中文, not 俄语.

File: src/i18n/ko/index.html [hunk 3]
```diff
@@ -34,17 +36,17 @@ <h2>불꽃 튀게 빠르고 설정이 필요 없는 웹 애플리케이션 번
...
-        </ul>
-      </div>
+      <form class="languages">
+        <select class="language-dropdown" onchange="location = this.options[this.selectedIndex].value;">
+          <option class="ko" value="https://ko.parceljs.org">🇰🇷 한국어</option>
+          <option class="en" value="https://parceljs.org">🇺🇸 English</option>
+          <option class="pt" value="https://pt.parceljs.org">🇧🇷 Português</option>
+          <option class="pl" value="https://pl.parceljs.org">🇵🇱 Polskie</option>
+          <option class="ru" value="https://ru.parceljs.org">🇷🇺 русский</option>
+          <option class="zh" value="https://zh.parceljs.org">🇨🇳 俄语</option>
```
Comment (dear-lizhihua) [inline src/i18n/ko/index.html]: should be 中文, not 俄语.

File: src/i18n/pl/index.html [hunk 4]
```diff
@@ -32,16 +34,17 @@ <h2>Oszałamiająco szybki, niewymagający konfiguracji program tworzący pakiet
...
-        </ul>
-      </div>
+      <form class="languages">
+        <select class="language-dropdown" onchange="location = this.options[this.selectedIndex].value;">
+          <option class="pl" value="https://pl.parceljs.org">🇵🇱 Polskie</option>
+          <option class="en" value="https://parceljs.org">🇺🇸 English</option>
+          <option class="pt" value="https://pt.parceljs.org">🇧🇷 Português</option>
+          <option class="ko" value="https://ko.parceljs.org">🇰🇷 한국어</option>
+          <option class="ru" value="https://ru.parceljs.org">🇷🇺 русский</option>
+          <option class="zh" value="https://zh.parceljs.org">🇨🇳 俄语</option>
```
Comment (dear-lizhihua) [inline src/i18n/pl/index.html]: should be 中文, not 俄语.

File: src/i18n/pt/index.html [hunk 5]
```diff
@@ -34,17 +36,17 @@ <h2>Empacotador de aplicações web ultra-rápido, sem configuração</h2>
...
-        </ul>
-      </div>
+      <form class="languages">
+        <select class="language-dropdown" onchange="location = this.options[this.selectedIndex].value;">
+          <option class="pt" value="https://pt.parceljs.org">🇧🇷 Português</option>
+          <option class="en" value="https://parceljs.org">🇺🇸 English</option>
+          <option class="ko" value="https://ko.parceljs.org">🇰🇷 한국어</option>
+          <option class="pl" value="https://pl.parceljs.org">🇵🇱 Polskie</option>
+          <option class="ru" value="https://ru.parceljs.org">🇷🇺 русский</option>
+          <option class="zh" value="https://zh.parceljs.org">🇨🇳 俄语</option>
```
Comment (dear-lizhihua) [inline src/i18n/pt/index.html]: should be 中文, not 俄语.

File: src/i18n/ru/index.html [hunk 6]
```diff
@@ -32,17 +34,17 @@ <h2>Молниеносно быстрый упаковщик для веб-пр
...
-        </ul>
-      </div>
+      <form class="languages">
+        <select class="language-dropdown" onchange="location = this.options[this.selectedIndex].value;">
+          <option class="ru" value="https://ru.parceljs.org">🇷🇺 русский</option>
+          <option class="en" value="https://parceljs.org">🇺🇸 English</option>
+          <option class="pt" value="https://pt.parceljs.org">🇧🇷 Português</option>
+          <option class="ko" value="https://ko.parceljs.org">🇰🇷 한국어</option>
+          <option class="pl" value="https://pl.parceljs.org">🇵🇱 Polskie</option>
+          <option class="zh" value="https://zh.parceljs.org">🇨🇳 俄语</option>
```
Comment (dear-lizhihua) [inline src/i18n/ru/index.html]: should be 中文, not 俄语.

File: src/i18n/zh/index.html [hunk 7]
```diff
@@ -42,17 +44,17 @@ <h2>极速零配置Web应用打包工具</h2>
...
-              <li><a href="https://pt.parceljs.org">🇧🇷</a></li>
-              <li><a href="https://ko.parceljs.org">🇰🇷</a></li>
-              <li><a href="https://pl.parceljs.org">🇵🇱</a></li>
-              <li><a href="https://ru.parceljs.org">🇷🇺</a></li>
-              <li><a href="https://es.parceljs.org">🇪🇸</a></li>
-            </ul>
-          </div>
+      <form class="languages">
+        <select class="language-dropdown" onchange="location = this.options[this.selectedIndex].value;">
+          <option class="zh" value="https://zh.parceljs.org">🇨🇳 俄语</option>
```
Comment (dear-lizhihua) [inline src/i18n/zh/index.html]: should be 中文, not 俄语.

**Pull request merged**